frontend/catalyst/api_extensions/quantum_operators.py

```diff
@@ -676,7 +774,7 @@ def qctrl_distribute(
     ops2 = []
     for op in tape.operations:
```

Comment:
One more change we could carry over from the adjoint impl is swapping `create_controlled_op` further down in this function to `qml.ctrl`.

The former always creates a PL Controlled instance, while the latter will create either a PL or Catalyst Controlled instance depending on context. Since the Catalyst version is now fully compatible the need for the existence of `create_controlled_op` is probably eliminated entirely.

Reply:
yeah, I tried with `ctrl` but it didn't work..

Reply:
But no! I tested before supporting generic PL's `Operator`! Clever idea @dime10 :+1: Let me try it out.

Reply:
Done! [cf70d1b](https://github.com/PennyLaneAI/catalyst/pull/771/commits/cf70d1b140e7417ff158dc5a43e930b053c17258)